src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -52,8 +57,6 @@ public static class ConnectionBuilder
     /// </summary>
     public class ConnectionBuilder<TSourceType>
     {
-        private bool _isUnidirectional;
-
         private bool _isBidirectional;
```

Comment:
I suggest to make it as computed, not a field.

Reply:
Computed from what?  Save/get from metadata?  Or are you thinking something else?

Reply:
From the current arguments. These properties are no more that reflection of some arguments set on the `FieldType`.

Reply:
after+first / before+last . Just check for args with taht name and type.